### PR TITLE
[zh] Sync web page for config service account

### PR DESCRIPTION
**Pull request by chenxuc**
This is for part of the items in the following umbrella issue:
[zh] Umbrella issue: pages out of sync in tasks section #26178

Service Account (L)

 content/zh/docs/tasks/configure-pod-container/configure-service-account.md

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -141,16 +149,30 @@ kind: ServiceAccount
...
 EOF
-serviceaccount/build-robot created
 ```
 
+<!--
+The name of a ServiceAccount object must be a valid
+[DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
+-->
+ServiceAccount 对象的名字必须是一个有效的
+[DNS 子域名](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-service-account.md:159]:
```suggestion
[DNS 子域名](/zh/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
```

File: content/zh/docs/tasks/configure-pod-container/configure-service-account.md [hunk 2]
```diff
@@ -384,23 +414,26 @@ myregistrykey
...
+* `--service-account-key-file`
 * `--service-account-signing-key-file`
-* `--service-account-api-audiences`
+* `--api-audiences`
+
 -->
 {{< note >}}
-ServiceAccountTokenVolumeProjection 在 1.12 版本中是 __beta__ 阶段，
-可以通过向 API 服务器传递以下所有参数来启用它：
+为了启用令牌请求投射，你必须在 `kube-apiserver` 声明以下的命令行参数：
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-service-account.md:430]:
```suggestion
为了启用令牌请求投射，你必须为 `kube-apiserver` 设置以下命令行参数：
```

**Comment by chenxuc** [comment]
@tengqm Thanks for the review. I've updated the commit.

**Review by ydFu — APPROVED** [review]

**Review by pigletfly — APPROVED** [review]

**Pull request merged**